applications/nrf5340_audio/tools/buildprog/buildprog.py

```diff
@@ -109,6 +110,10 @@ def __build_cmd_get(core: Core, device: AudioDevice, build: BuildType, pristine,
         if options.nrf21540:
             device_flag += " -DSHIELD=nrf21540ek_fwd"
 
+        if options.user_specific_name:
+            user_specific_bt_name = "NRF5340_AUDIO_" + getpass.getuser().upper()
```

Comment:
We need to truncate if the user name is too long.

Reply:
What is too long? The compiler compilains at total length of 248. I use a max length of 50 for now